tests/dbt_transformation_modify_mock_test.go

```diff
@@ -82,7 +81,7 @@ func prepareDbtTransformationModifyResponse() string {
 				"dbt_model_id": "%v",
 				"next_run": "%v",
 				"created_at": "%v",
-				"run_tests": "%v",
+				"run_tests": %v,
```

Comment:
Should we delete this from response?

Reply:
No, while real API still returns it we have to keep this to ensure it won't affect deserialization on client side